Solution1/ConsoleApp/Configuration/FileConfig.cs

```diff
@@ -12,5 +12,7 @@ public class FileConfig : IConfig
         public bool IsDebugMode => bool.TryParse(ConfigurationManager.AppSettings["isDebugMode"], out var value) ? value : default;
 
         public int? RequestTimeout => int.TryParse(ConfigurationManager.AppSettings["requestTimeout"], out var value) ? value : null;
+
+        public string FormatDateTime => ConfigurationManager.AppSettings["formatDateTime"];
```

Comment:
in console application you don't have any date formats, so why have you added this?